### PR TITLE
Pyperf meta updates

### PR DESCRIPTION
**Pull request by sousinha97**
Add header and meta information.

### CODE REVIEW

File: pyperf/pyperf_run [hunk 1]
```diff
@@ -78,6 +78,10 @@ generate_csv_file()
 	reduce=0
 	res_count=0
 	value_sum=0
+
+  $TOOLS_BIN/test_header_info --front_matter --results_file "${1}.csv" --host $to_configuration --sys_type $to_sys_type --tuned $to_tuned_setting --results_version $PYPERF_VERSION --test_name $test_name_run
+
+	echo "Test:Avg:Unit" > "${1}.csv"
```
Comment (dvalinrh) [inline pyperf/pyperf_run:84]: You are overwritting the front matter just outputted.

**Review by dvalinrh — CHANGES_REQUESTED** [review]

**Comment by dvalinrh** [comment]
LGTM

File: pyperf/pyperf_run (hunk 1 above)
Comment (sousinha97) [reply pyperf/pyperf_run]: Done

File: pyperf/pyperf_run [hunk 2]
```diff
@@ -297,6 +301,7 @@ if [ $to_pbench -eq 0 ]; then
 	else
 		echo Ran > test_results_report
 	fi
+
```
Comment (frival) [inline pyperf/pyperf_run:304]: It looks like this is spurious whitespace?

**Pull request merged**